custom-resources/cert-manager-issuer.tf

```
@@ -30,7 +30,7 @@ resource "kubernetes_manifest" "cluster_issuer" {
     "spec" = {
       "acme" = {
         // need to get this e-mail from the domain
-        "email" : "support+letsencrypt@massdriver.cloud"
+        "email" : "${var.md_metadata.contact_email}"
```

```suggestion
        "email" : var.md_metadata.contact_email
```
Don't need to do string interpolation.